pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
 	ServerSecretName string `json:"serverSecretName"`
-	// ClientSecretName contains etcd client key/cert, along with client CA cert.
+	// ClientSecretName contains etcd client key/cert and CA cert.
+	// This is used by operator.
```

Comment:
this is used by the operator to make outgoing https connections to member pod client interfaces